app/controllers/spree/admin/base_controller.rb

```diff
@@ -7,6 +7,7 @@ class BaseController < ApplicationController
       layout '/spree/layouts/admin'
 
       include I18nHelper
+      include Core::ControllerHelpers::Auth
```

Comment:
This is already included in the parent `ApplicationController`, so we shouldn't need this line.